layout/plugin/pjax.jsx

```diff
@@ -0,0 +1,15 @@
+const { Component, Fragment } = require('inferno');
+
+class Pjax extends Component {
+    render() {
+        const { helper } = this.props;
+        const { url_for, cdn } = helper;
+
+        return <Fragment>
+            <script src={cdn('pjax', '0.2.8', 'min/pjax.min.js')}></script>
```

Comment:
# Inconsistent CDN Format

My CDN setup: `cdn: '[cdnjs]https://lib.baomitu.com/${package}/${version}/${filename}'`

The URL outputted by `layout/common/scripts.jsx` is totally correct: `<script src="https://lib.baomitu.com/clipboard.js/2.0.4/clipboard.min.js" defer=""></script>`

However, the one generated from this line: `<script src="https://lib.baomitu.com/pjax/0.2.8/min/pjax.min.js"></script>` clearly deviates from the expected format. How should I go about fixing it?

Reply:
Okay, it turns out does not have `min/`.